scripts/zkai_d128_gate_value_compact_preprocessed_gate.py

```
@@ -0,0 +1,626 @@
+        "baseline_profile_row",
+        "mutations_checked",
+        "mutations_rejected",
+        "all_mutations_rejected",
+        "payload_commitment",
+    }
+    if not allow_missing_mutation_summary:
+        expected_fields.add("mutation_cases")
+    if set(payload) != expected_fields:
+        raise GateValueCompactGateError(f"payload field drift: {sorted(set(payload) ^ expected_fields)}")
+    if payload["schema"] != SCHEMA:
+        raise GateValueCompactGateError("schema drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)